ipu-plugin/pkg/ipuplugin/lifecycleservice.go

```diff
@@ -237,6 +237,45 @@ func getFilteredPFs(pfList *[]netlink.Link) error {
 	return nil
 }
 
+/*
+If IDPF net devices dont show up on host-side, this function is a best effort to
```

Comment:
Maybe add the case when IDPF netdevs don't show up?

Reply:
@saynb Done. Updated comment to explain the case where we could have this issue.